Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -279,6 +279,24 @@ class CreateTemplateString: Operation {
     }
 }
 
+class CallTaggedTemplate: Operation {
```

Comment:
Nit: maybe this should be called `CreateTaggedTemplateString` to be consistent with `CreateTemplateString`?